frontend/src/helper/Controller.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import axios from "./axios";
+
+export const getAllMetalPrices = async () => {
+    const priceList = await axios.get("/prices");
+    return priceList
+};
+
+
+export const getListOfAllExchangeRates = async () => {
+    const currencyRates = await axios.get("/getListOfAllExchangeRates");
+    return currencyRates
+};
+
+export const getAuthentication = async (username, password) => {
+    return await axios.post("/getAuthentication", {
+        username,
+        password,
+    });
+}
+
+export const getAnalysisSummaryList = async () => {
+    const HEADERS = {
+        "authentication": localStorage.getItem("token")
+    }
```

Comment:
maybe extract some method here
```suggestion
    const headers = getAuthenticationHeaders()
```